packages/larva-patterns/components/c-link/c-link.twig

```diff
@@ -1,3 +1,3 @@
-<a class="c-link {{ modifier_class }} {{ c_link_classes }}" href="{{ c_link_url }}">
+<a class="c-link {{ modifier_class }} {{ c_link_classes }}" href="{{ c_link_url }}" {% if c_link_is_external %} target="_blank" rel="noopener" {% endif %}>
```

Comment:
I agree we should support these attributes! At this point in the system maturity, we also want to limit the creation of special-case names – does this work instead, and then we can keep to the language already in the system:
```
 <a class="c-link {{ modifier_class }} {{ c_link_classes }}" href="{{ c_link_url }}" {% if c_link_target_attr %} target="{{ c_link_target_attr }}" {% endif %} {% if c_link_rel_attr %} rel="{{ c_link_rel_attr }}" {% endif %}>
```
And in order to enforce the use of both attributes for the performance benefits, there can be a `c-link.external.js` variation. Then we can keep the naming to the variant level of abstraction rather than adding it to the language. There is not currently a good way to have visibility into all of these variants, but there will be!

Reply:
I'm OK with this, there's no need to enforce this with the external variation. I'll update the code.

Reply:
@mehigh Before I merge this, can you confirm this parses correctly into PHP?

Reply:
npm run parse yielded no errors on my end, so I can assume it's correct (I didn't check this in the WordPress integrated templates context though, but should be fine).

Reply:
@mehigh @laras126 I wasn't able to test the update inside this repo (coulnd't quickly figure out how to run parse in here).

But I tested with a copy in the theme and it's parsing as expected:

    <?php
    // This is a generated file. Refer to the relevant Twig file for adjusting this markup.
    ?>
    <a class="c-link <?php echo esc_attr( $modifier_class ?? '' ); ?> <?php echo esc_attr( $c_link_classes ?? '' ); ?>" href="<?php echo esc_url( $c_link_url ?? '' ); ?>" <?php if ( ! empty( $c_link_target_attr ) ) { ?> target="<?php echo esc_attr( $c_link_target_attr ?? '' ); ?>" <?php } ?> <?php if ( ! empty( $c_link_rel_attr ) ) { ?> rel="<?php echo esc_attr( $c_link_rel_attr ?? '' ); ?>" <?php } ?>>
        <?php echo esc_html( $c_link_text ?? '' ); ?>
    </a>

Reply:
Thanks all! Adding this to docs:
To test the parsing of patterns in larva-patterns:
- In a clone of this repository, cd into packages/larva
- Run `npm run parser` and confirm manually that it is correct

(we need automated testing for this)